src/ASM/LR/ASMu2Dmx.C

```diff
@@ -1011,8 +1011,7 @@ bool ASMu2Dmx::refine (const LR::RefineData& prm, Vectors& sol)
     // Uniformly refine to find basis 1
     if (ASMmxBase::Type == ASMmxBase::SUBGRID) {
       m_basis[0].reset(refBasis->copy());
-      if (!ASMmxBase::subgridH)
-        projBasis = m_basis[0];
+      projBasis = m_basis[0];
```

Comment:
Be consistent at least (either `[0]` or `.front()`, not a "salig blanding"), for the readability.